### ISSUE TITLE
Timeout Issue with get_indicator_details_full

### ISSUE THREAD

**Issue by baalgar** [AlienVault-OTX/OTX-Python-SDK#49]
When passing in anything other than URLs like hashes into otx.get_indicator_details_full for type IndicatorTypes.URL it hangs and there's no way to control timeout

**Comment by rustybrooks** [comment]
Yeah so what's actually happening here is that it's retrying over and over with increasing timeouts, because it's getting a 500 from the server.  Probably:
a) we shouldn't retry on 500 errors 
b) we should fix the 500 error - it should probably give a 400 instead
I will fix b) right away, a) will go out in the next release.

**Comment by baalgar** [comment]
Thanks, I'll try that out. I temporarily bypassed that issue by actively parsing indicator types as they come in and then doing something like eval(IndicatorTypes.type)
